serverless/install/install-serverless-operator.adoc

```diff
@@ -13,6 +13,11 @@ This guide walks cluster administrators through installing the {ServerlessOperat
 {ServerlessProductName} is supported for installation in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
 ====
 
+[NOTE]
+====
+{ServerlessProductName} currently cannot be used in a multi-tenant configuration on a single cluster.
+====
+
```

Comment:
We shouldn't have two notes right next to each other like this - maybe these two could be combined into a section with a title, something like "Supported configurations" or something and list them as bullets?